src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -282,21 +317,6 @@ class AccountControllerTest @Autowired constructor(
                 fun `should be in the past`() = validationTester.isPastDate()
             }
 
-            @NestedTest
-            @DisplayName("photoPath")
-            inner class PhotoPathValidation {
-                @BeforeAll
-                fun setParam() {
-                    validationTester.param = "photoPath"
-                }
-
-                @Test
-                fun `should be null or not blank`() = validationTester.isNullOrNotBlank()
```

Comment:
Why can't this be tested?

Reply:
We don't include this param in the request